### DISCUSSION TITLE
Encourage sign-ups with business email addresses

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#5495] [Ideas]
As a team collaboration tool, it's not surprising to learn that most users sign-up with a business email address, and that those that do, are more likely to participate in a meeting and continue to participate in meetings. 

If we see a user enter a personal email address when signing up, we should encourage or suggest that they consider signing up with a business email address. 

I'm imagining a prompt that says, "🤔  Hmmm that looks like a personal address - if you're trying out Parabol with your team mates, we recommend using your professional email address." 

This wouldn't block anyone from signing up with a personal address, just encourage them to signup with the business one.

**Comment by ackernaut** [comment]
I like this idea. Would this pop up once they’ve entered a personal email in the field?

I wonder if there’s a use case around folks wanting (inevitably) to change their email address to their work email address. This could be an option when managing their account, and perhaps any changes to email require email verification?

**Comment by avivapinchas** [comment]
> Would this pop up once they’ve entered a personal email in the field?

That's what I was thinking! 

> I wonder if there’s a use case around folks wanting (inevitably) to change their email address to their work email address. 

This could be an option when managing their account, and perhaps any changes to email require email verification?
Probably! I imagine there's a few cases where folks use a personal email address: 

1.  They're doing some kind of side-hustle or project on the side
2. They want to try out Parabol and aren't sure about restrictions with their company email or needed approvals 
3. They accidentally use it, either due to auto-fill, moving quickly, or signing in with Google and not paying attention 

I think these two tactics would possibly help with the third scenario, but not the first two.

**Reply by ackernaut** [reply]
I see, I see
